crates/wasmcloud-control-interface/src/broker.rs

```diff
@@ -64,7 +64,7 @@ pub mod commands {
 pub mod queries {
     use super::prefix;
 
-    pub fn linkdefinitions(nsprefix: &Option<String>) -> String {
+    pub fn link_definitions(nsprefix: &Option<String>) -> String {
```

Comment:
Since this is API breaking, it requires a minor build bump in the `Cargo.toml` file.